frontend/src/pages/premium.page.tsx

```diff
@@ -68,11 +70,15 @@ const PremiumPromo: NextPage = () => {
           </div>
         </section>
 
-        {isBundleAvailableInCountry(runtimeData.data) && (
+        {isMegabundleAvailableInCountry(runtimeData.data) ? (
+          <section id="vpn_promo" className={styles["bundle-banner-section"]}>
```

Comment:
suggestion (blocking): change `id` value.
```suggestion
          <section id="megabundle_promo" className={styles["bundle-banner-section"]}>
```